src/main/java/software/amazon/awssdk/crt/http/HttpClientConnectionManagerOptions.java

```diff
@@ -198,23 +201,25 @@ public HttpClientConnectionManagerOptions withProxyOptions(HttpProxyOptions prox
     public HttpProxyOptions getProxyOptions() { return proxyOptions; }
 
     /**
-     * If set to true, then the TCP read back pressure mechanism will be enabled. You should
-     * only use this if you're allowing http response body data to escape the callbacks. E.g. you're
-     * putting the data into a queue for another thread to process and need to make sure the memory
-     * usage is bounded (e.g. reactive streams).
-     * If this is enabled, you must call HttpStream.UpdateWindow() for every
-     * byte read from the OnIncomingBody callback.
      * @return true if manual window management is used, false otherwise
+     * @see #withManualWindowManagement
      */
     public boolean isManualWindowManagement() { return manualWindowManagement; }
 
     /**
-     * If set to true, then the TCP read back pressure mechanism will be enabled. You should
+     * If set to true, then you must manage the TCP read backpressure mechanism. You should
```

Comment:
`the TCP read backpressure` is that still true? we actually have the connection manager support both h2 and h1.

Reply:
good point. I'll edit